manifests/init.pp

```diff
@@ -1,64 +1,94 @@
-# == Class: ca_cert
+# @summary
+#   This module manages the user defined certificate authority (CA)
+#   certificates on the server. On OSes that support a distrusted
+#   folder the module also manages distrusting system default CA certificates.
 #
-# This module manages the user installed certificate authority (CA)
-# certificates installed on the server. It does not manage operating
-# system defaults CA certificates.
+# @example Basic usage
+#   class { 'ca_cert': }
 #
-# === Parameters
+#   class { 'ca_cert':
```

Comment:
you need another `@example something something` above each codeblock.

Reply:
Hmm, `manage_all_user_CAs` is not even a param any more ...
I'll fix this :)